workflow/cron/controller.go

```diff
@@ -205,7 +205,7 @@ func (cc *Controller) addCronWorkflowInformerHandler() {
 }
 
 func (cc *Controller) syncAll() {
-	log.Info("Syncing all CronWorkflows")
+	log.Debug("Syncing all CronWorkflows")
```

Comment:
Change to `Debug` to avoid spamming the logs

Reply:
why do we log every 10s?